e2e/stories/jfs.e2e.ts

```diff
@@ -37,6 +37,23 @@ describe('Fileset Module', () => {
     });
   });
 
+  it('should upload multiple files', (done) => {
+    const paths = [
+      'e2e/resources/badge-facebook.png',
+      'e2e/resources/badge-linkedin.png',
+      'e2e/resources/badge-twitter.png',
+    ];
+    const files = paths.map((path) => ({ file: fs.createReadStream(path) }));
+    let filesUploaded = 0;
+    jfs.fileset(filesetName).upload(files).subscribe((result) => {
+      joiAssert(result, FilesetRecordSchema);
+      filesUploaded++;
+    }, done, () => {
```

Comment:
wouldn't `done` here make the test succeed in case of error?

Reply:
no, if you call done() with an argument (which will happen in case of an error), the test will fail